core/src/main/java/org/apache/iceberg/InheritableMetadataFactory.java

```diff
@@ -78,6 +73,13 @@ public <F extends ContentFile<F>> ManifestEntry<F> apply(ManifestEntry<F> manife
         manifestEntry.setFileSequenceNumber(sequenceNumber);
```

Comment:
not caused by this PR. can we fix the typo in the comment? `iff` -> `off`

Reply:
I believe `iff` is correct and stands for `if and only if`

Reply:
"iff" here means "if and only if", I think. It's a common short-hand.